### DISCUSSION TITLE
.NET 9 RC 1 images now available

### DISCUSSION THREAD

**Discussion by lbussell** [dotnet/dotnet-docker#5873] [Announcements]
Container images for .NET 9 Release Candidate 1 are now available. This is the first of two release candidates for .NET 9. .NET 9 RC 1 has a Go-Live support license, so you can confidently use it in production.

The existing `9.0-preview` tags will be replaced with `9.0` going forwards. The following repos have been updated:

* dotnet/sdk - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/sdk/about)
* dotnet/aspnet - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/aspnet/about)
* dotnet/runtime - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/runtime/about)
* dotnet/runtime-deps - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/runtime-deps/about)
* dotnet/monitor - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/monitor/about)
* dotnet/monitor/base - [Microsoft Artifact Registry](https://mcr.microsoft.com/product/dotnet/monitor/base/about)

For more info, check out the [.NET 9 RC 1 release announcement](https://github.com/dotnet/core/discussions/9496) and see [what's new in .NET 9](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-9/overview).

**Comment by akoken** [comment]
Where is the `jammy-chiselled` tag? Can't use the Noble distro due to the issue discussed [here](https://github.com/dotnet/dotnet-docker/discussions/5493).

**Reply by MichaelSimons** [reply]
.NET will only be offering Ubuntu Noble images per the [supported platforms policy](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-platforms.md#linux):

> Ubuntu — support the latest LTS version at the time a major.minor version of .NET is released. As new LTS versions are released, support is added to the latest .NET version and latest LTS (if they differ).

**Reply by akoken** [reply]
But it simply doesn’t work. I want to use a distroless image, but the only available option is failing. Am I supposed to create and maintain my own jammy-based image now?

**Reply by akoken** [reply]
Never mind, I found a workaround, though it’s not ideal.
